cmd/podman/containers/rm.go

```diff
@@ -69,9 +71,13 @@ func rmFlags(cmd *cobra.Command) {
 	flags.BoolVarP(&rmOptions.Volumes, "volumes", "v", false, "Remove anonymous volumes associated with the container")
 
 	cidfileFlagName := "cidfile"
-	flags.StringArrayVar(&cidFiles, cidfileFlagName, nil, "Read the container ID from the file")
+	flags.StringArrayVar(&rmCidFiles, cidfileFlagName, nil, "Read the container ID from the file")
```

Comment:
`killCidFiles` is initialized with `[]string{}` not `nil`.  Is there any noticeable difference?

Reply:
I think, there is not functional difference between `[]string{}` and `nil`.
I'll unify the arguments to nil.